content/sponsors/gold/geocat.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "GeoCat bv"
+page_header_bg: "images/background/page-title-bg.jpg"
+date: 2024-01-14T20:22:24+02:00
+designation: "Platinum"
+image: "images/sponsors/geocat-platinum.png"
+draft: false
+type: "sponsor"
+layout: "single"
+social:
+- name: "Homepage"
+  icon: "fa-solid fa-house"
+  link: "https://www.geocat.net/"
+- name: "mastodon"
+  icon: "fa-brands fa-mastodon"
+  link: "https://fosstodon.org/@geocat"
+- name: "threads"
+  icon: "fa-brands fa-threads"
+  link: "https://www.threads.net/@geocat_bv"
+- name: "linkedin"
+  icon: "fa-brands fa-linkedin"
+  link: "https://www.linkedin.com/company/geocat-bv/"
+- name: "instagram"
+  icon: "fa-brands fa-instagram"
+  link: "https://www.instagram.com/geocat_bv/"
+---
+
+GeoCat provides advanced software and services that make publishing geographic and open
+ data on the internet easier and more efficient than ever. Our expert developers play a leading 
+role in open-source software, creating sustainable applications based on Open Standards and 
+the Free and Open Source software philosophy.
+
+Trusted by organizations such as the Dutch Cadastre (National Geo Register), Rijkswaterstaat,
+ the Ministry of Economic Affairs and Agriculture, the Ministry of Environment and Climate Change Canada
+ (ECCC), multiple national portals in and outside of Europe, and the European INSPIRE geoportal,
+ our solutions play a key role in building robust dataspace infrastructures.
+
+Our product suite includes GeoCat Find (powered by GeoNetwork, the open-source catalog for 
+geographic and open data), GeoCat Map (built on GeoServer), and GeoCat Bridge (seamlessly 
+publishing data from your desktop GIS with a single click). For an even more streamlined experience,
+ our solutions are available via our cloud platform, GeoCat Live.
+
+We also offer expert consultancy and training to help you get the most out of our products.
+
+Want to learn more? Contact us at [info@geocat.net](mailto:info@geocat.net) or visit [www.geocat.net](https://www.geocat.net).

```

Comment:
use `extlink` shortcode (i.e. `{{ extlink href="https://www.geocat.net" title="www.geocat.net" }}` here) for any external links so they open in a new browser window.